client/src/App.jsx

```diff
@@ -5,8 +5,12 @@ import Videos from './components/Videos/Videos.jsx';
 import Comments from './components/Comments/Comments.jsx';
 import Login from './containers/LogIn/Login.jsx';
 import Main from './containers/Main/Main.jsx';
+<<<<<<< HEAD
```

Comment:
Does this "<<<<<<<< HEAD" actually show up in your source code? Let me know before I approve.